packages/api/src/controllers/helpers.ts

```diff
@@ -674,7 +674,7 @@ export const triggerCatalystPullStart =
               res.status
             } error=${JSON.stringify(body)}`
           );
-          await sleep(1000);
+          await sleep(250);
```

Comment:
I wonder if this could have bigger risks of concurrency issues, given each of these requests might hit a different catalyst instance. Then we could have multiple instances starting a pull at the same time (not sure what errors could happen after that tho). Let's keep this in mind if we notice any issues (this has been deployed to prod earlier today)